xwiki-platform-core/xwiki-platform-repository/xwiki-platform-repository-server-ui/src/main/resources/ExtensionCode/ExtensionSheet.xml

```diff
@@ -242,6 +242,7 @@
     |Source|$doc.display('source')
     |Display Icon Location
 (Syntax: #set($comment = '##')${comment}icon:{{html}}&lt;a href="http://www.famfamfam.com/lab/icons/silk/previews/index_abc.png" target="_blank"&gt;icon name&lt;/a&gt;{{/html}}${comment})|$doc.display("icon")
+    |Show installed count|#if($proxyExtensionObject)$doc.display('showInstalledCount', 'view')#else$doc.display('showInstalledCount')#end~
```

Comment:
installed count has nothing to do with the proxy object. proxy object is for imported extension and you don't need the extension to be imported for it to be installed.